crates/miden-lib/asm/miden/contracts/wallets/aux.masm

```diff
@@ -0,0 +1,25 @@
+use.miden::tx
+
+#! Adds the specified asset to the output note with the specified index.
+#!
+#! This procedure is expected to be invoked using a `call` instruction. It makes no guarantees about
+#! the contents of the `PAD` elements shown below. It is the caller's responsibility to make sure
+#! these elements do not contain any meaningful data.
+#!
+#! Inputs:  [ASSET, note_idx, pad(11)]
+#! Outputs: [ASSET, note_idx, pad(11)]
+#!
+#! Where:
+#! - note_idx is the index of the output note.
+#! - ASSET is the fungible or non-fungible asset of interest.
+#!
+#! Panics if:
+#! - the fungible asset is not found in the vault.
+#! - the amount of the fungible asset in the vault is less than the amount to be removed.
+#! - the non-fungible asset is not found in the vault.
+#!
+#! Invocation: call
+export.add_asset_to_note
+    exec.tx::add_asset_to_note
+    # => [ASSET, note_idx, pad(11) ...]
```

Comment:
```suggestion
    # => [ASSET, note_idx, pad(11)]
```

Nit: Personally, I don't use `...` in MASM stack comments, but can't recall if we've agreed on this - we probably should have a consensus. I think `...` is supposed to say that there could be more elements on the stack, but that is practically always the case when `exec`-ing, so in that case, I think it's unnecessary.

In the context of `call` like here, we explicitly describe the number of `pad` elements that fill up the stack to 16, so it also seems unnecessary here.

Any thoughts @bobbinth or @Fumuran?

If we agree not to use them, we could clean up a bunch of comments in the repo and in any case we might want to explicitly describe this in `crates/miden-lib/masm_doc_comment_fmt.md`.